include/aspect/material_model/rheology/visco_plastic.h

```diff
@@ -249,6 +249,12 @@ namespace aspect
            */
           bool allow_negative_pressures_in_plasticity;
 
+          /**
+           * Whether to use adiabatic pressure instead of full pressure (default)
+           * when calculating creep (diffusion, dislocation, and peierls) viscosity.
```

Comment:
Expand the documentation here as to why this option may be useful (see related comments below).